wordpress-mobile/WordPress-iOS
# Turn on Muriel colors 🎉

Refs #11683

Enables Muriel colors for all users

To test:
- I'm sure you can eyeball the changes, but you could build for release or something and see if it looks all Muriel-ized

Update release notes:
